server/libs/domain/src/share/share.service.ts

```diff
@@ -57,4 +61,9 @@ export class ShareService {
     const link = await this.shareCore.save(authUser.id, id, dto);
     return mapSharedLink(link);
   }
+
+  async getAllPartners(authUser: AuthUserDto): Promise<PartnerResponseDto[]> {
+    const partners = await this.partnerCore.getAll(authUser.id, 'shared-with');
+    return partners.map(mapPartner);
+  }
```

Comment:
Just a note - if this was moved to the `PartnerService`, all the partner logic would be accessed from a single controller/service and `PartnerCore` could be merged/flattened back into the `PartnerService`.

Reply:
`PartnerCore` seems to be necessary by `AssetService` to check asset access

Reply:
Yes, this looks like one method which makes sense for another controller to potentially handle.